aimmo-game/simulation/world_map.py

```diff
@@ -221,7 +234,8 @@ def _reset_score_locations(self, num_avatars):
             cell.generates_score = True
 
     def _add_pickups(self, num_avatars):
-        target_num_pickups = int(math.ceil(num_avatars * self.settings['TARGET_NUM_PICKUPS_PER_AVATAR']))
+        target_num_pickups = int(math.ceil(num_avatars *
```

Comment:
if you make a variable for the `num_avatars * self.settings['TARGET_NUM_PICKUPS_PER_AVATAR']` it might be a better way of splitting it up

Reply:
I kind of lost consistency here, I'll commit it like the rest of the file does it and try to keep this standard. Comment if you still don't like it ;p

![](https://cdn.pbrd.co/images/GU5yFtP.png)